### ISSUE TITLE
Chemical formula

### ISSUE THREAD

**Issue by gustavojra** [FermiQC/Molecules.jl#9]
A function that takes an array of atoms and return a string representation of the molecule is desired for printing purposes!

```
julia> chemical_formula(atoms)
H2O
```

Note: there are symbols for lower case numbers. See `BasisFunction`

**Comment by Anna-Frerichs** [comment]
Is this still relevant? I am happy to have a look

**Comment by gustavojra** [comment]
Yes, that is a small little task but could be fun for someone who's looking for something simple
